lit/tests/Property/Good.sc

```diff
@@ -76,15 +76,24 @@ args.head match {
     // CHECK-LABEL: OMReferenceTarget:~PropertyTest|PropertyTest>i
     println(obj.field("p").asInstanceOf[PanamaCIRCTOMEvaluatorValuePath].toString)
 
-    // CHECK-NEXT: .a => { [ [ omInteger{123} ] ] }
-    // CHECK-NEXT: .b => { [ [ omInteger{456} ] ] }
-    // CHECK-NEXT: .p => { path{OMReferenceTarget:~PropertyTest|PropertyTest>i} }
+    // CHECK-NEXT: .a => { [ [ 123 ] ] }
+    // CHECK-NEXT: .b => { [ [ 456 ] ] }
+    // CHECK-NEXT: .p => { OMReferenceTarget:~PropertyTest|PropertyTest>i }
     obj.foreachField((name, value) => println(s".$name => { ${value.toString} }"))
 
     // CHECK-NEXT: module{_1_Anon}
     // CHECK-NEXT: module{PropertyTest_Anon}
     // CHECK-NEXT: module{PropertyTest}
     converter.foreachHwModule(name => println(s"module{$name}"))
+
+    val aInt = obj.field("a")
+      .asInstanceOf[PanamaCIRCTOMEvaluatorValueList]
+      .getElement(0)
+      .asInstanceOf[PanamaCIRCTOMEvaluatorValueList]
+      .getElement(0)
+      .asInstanceOf[PanamaCIRCTOMEvaluatorValuePrimitiveInteger]
+      .integer
+    assert(123 == aInt)
```

Comment:
It's a dilemma to get these type since Scala doesn't know the real type.
In the following PR, we may need to think about how to safely get these Scala type.
think about provide these API to end users in the future:
https://github.com/com-lihaoyi/upickle/blob/3845e9ef57851075756b3d32efe3898e2dc13391/ujson/src/ujson/Value.scala#L17-L89